### PR TITLE
.github: Add a "Module requests" issue template

### PR DESCRIPTION
**Pull request by Pandapip1**
## Description of changes

Self-explanitory. I've seen several issues with this naming scheme, so it seems to make sense to formalize it.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/module_request.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: Module requests
+about: For packages that are missing
```
Comment (FliegendeWurst) [inline .github/ISSUE_TEMPLATE/module_request.md:3]:
```suggestion
about: For modules that you would like to see
```

File: .github/ISSUE_TEMPLATE/module_request.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title: 'Module request: MODULENAME'
+labels: '9.needs: module (new)'
+assignees: ''
+
+---
+
+**Description**
+
+<!-- Describe what the module should accomplish: -->
+
```
Comment (FliegendeWurst) [inline .github/ISSUE_TEMPLATE/module_request.md:13]: I would add a note to tag the maintainers of affected packages.

**Review by FliegendeWurst — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/module_request.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+name: Module requests
+about: For modules that you would like to see
+title: 'Module request: MODULENAME'
+labels: '9.needs: module (new)'
+assignees: ''
+
+---
+
+## Description
```
Comment (Scrumplex) [inline .github/ISSUE_TEMPLATE/module_request.md:10]: Other templates use `###` for sections

File: .github/ISSUE_TEMPLATE/module_request.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+name: Module requests
+about: For modules that you would like to see
```
Comment (Atemu) [inline .github/ISSUE_TEMPLATE/module_request.md:3]: Nit:

```suggestion
about: For NixOS modules that you would like to see
```

File: .github/ISSUE_TEMPLATE/module_request.md (hunk 3 above)
Comment (Pandapip1) [reply .github/ISSUE_TEMPLATE/module_request.md:10]: This is true, although semantically a second-level heading would make more sense. I'll switch this to a third-level heading and submit a separate PR to change to second-level headings.

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**